README.md

```diff
@@ -25,141 +27,106 @@ a recommended tool for more full-featured analysis of JFR files beyond what
-behaviour. `CRYOSTAT_TARGET_CACHE_MAX_CONNECTIONS` specifies how many
-connections may be held open at once. `-1` may be used to leave the cache size
-unlimited.  `CRYOSTAT_TARGET_CACHE_TTL` specifies how long (in seconds) these
-connections will be cached before they are closed due to inactivity.
-
-For logging, Cryostat uses SLF4J with the java.util.logging binding.
-The default configuration can be overridden by mounting the desired
-configuration file in the container, and setting the environment variable
-`CRYOSTAT_JUL_CONFIG` to the path of that file.
-
-Some of Cryostat's dependencies also use java.util.logging for their logging.
-Cryostat disables
-[some of these](https://github.com/cryostatio/cryostat-core/tree/main/src/main/resources/config/logging.properties)
-by default, because they generate unnecessary logs.
-However, they can be reenabled by overriding the default configuration file
-and setting the disabled loggers to the desired level.
+
+### Development Run Outside of Containers
+* `MAVEN_OPTS="-Dcom.sun.management.jmxremote.port=9091 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.autodiscovery=true" mvn clean prepare-package exec:java`.
+
+### Run on Kubernetes/Openshift
+* See the [cryostat-operator](https://github.com/cryostatio/cryostat-operator)
+
+### Run on local podman
+* `run.sh`
+
+### Run on local podman with Grafana, jfr-datasource and demo application
+* `smoketest.sh`
+
+## CONFIGURATION
+
+Cryostat can be configured via the following environment variables
+
+#### Configuration for cryostat-web
+
+* `CRYOSTAT_WEB_HOST`: the hostname used by cryostat-web
+* `CRYOSTAT_WEB_PORT`: the internal port used by cryostat-web
+* `CRYOSTAT_EXT_WEB_PORT`: the external port used by cryostat-web
+* `CRYOSTAT_CORS_ORIGIN`: the origin for CORS to load a different cryostat-web instance
+
+#### Configuration for cryostat
+
+* `CRYOSTAT_MAX_WS_CONNECTIONS`: the maximum number of webscoket client connections allowed (minimum 1, maximum 64, default 2)
+* `CRYOSTAT_AUTH_MANAGER`: the authentication/authorization manager used for validating user accesses. See the `USER AUTHENTICATION / AUTHORIZATION` section for more details. Set to the fully-qualified class name of the auth manager implementation to use, ex. `io.cryostat.net.BasicAuthManager`.
+* `CRYOSTAT_PLATFORM`: the platform client  used for performing platform-specific actions, such as listing available target JVMs. If `CRYOSTAT_AUTH_MANAGER` is not specified then a default auth manager will also be selected corresponding to the platform, whether that platform is specified by the user or automatically detected. Set to the fully-qualified name of the platform detection strategy implementation to use, ex. `io.cryostat.platform.internal.KubeEnvPlatformStrategy`.
+* `CRYOSTAT_CONFIG_PATH`: the local filesystem path for the configuration directory (default `/opt/cryostat.d/conf.d`)
```

Comment:
Should this explain that "local" means within the container, ie local to the Cryostat process?

Reply:
An interesting question. When I think of local in the context of containers, I do mean local to the container already heh :P But then local on non-containers is the host machine. I'll try to think of something that's generically less confusing.

Reply:
After further investigation, local in the context of containers is used to refer to the host machine filesystem. I guess that was just me. I've removed that confusing term for now.